templates/journald.conf.j2

```diff
@@ -0,0 +1,44 @@
+# SPDX-License-Identifier: MIT
+
+{{ ansible_managed | comment }}
+[Journal]
+{# Persistent journal #}
+{% if journald_persistent is true %}
```

Comment:
```suggestion
{% if journald_persistent | bool %}
```
Ansible doesn't like comparisons to `true` and `false` - it's already a boolean, just use it in a boolean context - the `| bool` is to ensure that the variable is a boolean - since it can be set by the user and the user might set it to `"true"` or `"something"`